### DISCUSSION TITLE
Error: Parse login request

### DISCUSSION THREAD

**Discussion by fourcandy** [bedrock-tool/bedrocktool#57] [Q&A] [answered]
whenever I try to join using my local ip to download worlds from the hive, it gives me this error

> error: parse login request: validate token 0: square/go-jose/jwt: validation failed, token not valid yet (nbf)

How do I fix it?

**Comment by olebeck** [comment, marked as answer]
your system time may be set wrong

**Reply by fourcandy** [reply]
Oh, now that I realise, my PC changes its time to somewhere in the US on every startup. Anyway, I fixed it by using the IP 127.0.0.1 instead of my local IP.
